### PR TITLE
feat: create themes page 

### PR DESCRIPTION
**Pull request by bityutskiyAO**

### CODE REVIEW

**Comment by bityutskiyAO** [comment]
Base branch for Themes new feature, will be merged after all subtasks are done

File: src/components/Themes/ThemeSection/ThemeSection.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import './ThemeSection.scss';
+
+export interface ThemeSectionProps {
+    sectionType: ThemeSectionType;
+}
+
+const b = block('theme-section');
+
+//TODO: need to create section logic
+const config = {
```
Comment (dgaponov) [inline src/components/Themes/ThemeSection/ThemeSection.tsx:15]: It looks better this way. What do you think?

```
const config: Record<string, React.ComponentType> = {
    [ThemeSectionType.Colors]: SomeComp,
    [ThemeSectionType.Typography]: SomeComp,
    [ThemeSectionType.BorderRadius]: SomeComp,
    [ThemeSectionType.Preview]: SomeComp,
}

export const ThemeSection = ({sectionType}: ThemeSectionProps) => {
    const Component = config[sectionType];

    return (
        <div className={b()}>
            <Component />
        </div>
    );
};
```

File: src/components/Themes/Themes.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    return (
+        <Grid className={b()}>
+            <Row className={b('title')}>
+                <Col sizes={12}>
+                    <Text className={b('title__text')}>{t('title')}</Text>
+                </Col>
+            </Row>
+            <Flex className={b('header-actions')}>
+                <Tags
```
Comment (dgaponov) [inline src/components/Themes/Themes.tsx:56]: In the design, the active tag has a different color.
image

File: src/components/Themes/Themes.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    className={b('tabs')}
+                    items={tags}
+                    value={activeTag}
+                    onChange={setActiveTag}
+                />
+
+                <Button
+                    className={b('export-theme-btn')}
```
Comment (dgaponov) [inline src/components/Themes/Themes.tsx:64]: In the design, the export button tag has a different border radius.

image

Comment (bityutskiyAO) [reply src/components/Themes/Themes.tsx]: all XL btn in landing has bigger border radius, i will go to Yuri and ask him about it

File: src/components/Themes/Themes.tsx (hunk 2 above)
Comment (bityutskiyAO) [reply src/components/Themes/Themes.tsx]: Снимок экрана 2024-06-17 в 12 21 35
I don't see diff..

File: src/components/Themes/ui/MainSettings/MainSettings.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        name,
+        theme,
+    });
+
+    return (
+        <ColorPickerInput
+            value={color}
+            defaultValue={color}
+            onChange={setColor}
+            // Цвет фона применяется к карточке, в которой он отображается. Чтобы можно было увидеть preview оборачиваем блок в border
```
Comment (imsitnikov) [inline src/components/Themes/ui/MainSettings/MainSettings.tsx:43]: use english in comments plz

**Review by imsitnikov — APPROVED** [review]

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**